src/Symfony/Component/Amqp/RetryStrategy/RetryStrategyInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Component\Amqp\RetryStrategy;
+
+interface RetryStrategyInterface
```

Comment:
Nevermind, GitHub didn't display the content of this class... That's why I missed a lot of things. Now it displays well and answers my comments.